src/lib/pipes/truncate-deso-address.pipe.ts

```diff
@@ -0,0 +1,20 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { GlobalVarsService } from 'src/app/global-vars.service';
+
+@Pipe({
+  name: 'truncateAddress',
+})
+export class TruncateAddressPipe implements PipeTransform {
+  constructor(public globalVars: GlobalVarsService) {}
```

Comment:
```suggestion
  constructor() {}
```

Reply:
Ended up taking out the whole constructor definition instead